walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,70 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.Set;
+import java.util.List;
+import java.util.ArrayList;
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
```

Comment:
maybe change the function name to be a little clearer in terms of what is actually happening. Maybe findAllAvailableTimeslots? It would be helpful as well to add javadoc that describes each of the parameters and describes what the function should return.